Core/src/org/sleuthkit/autopsy/geolocation/datamodel/GeoLocationDataException.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.autopsy.geolocation.datamodel;
+
+/**
+ *
+ * 
+ */
+public class GeoLocationDataException extends Exception {
+
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Default constructor when error message is not available
+     */
+    public GeoLocationDataException() {
+        super("No error message available.");
+    }
```

Comment:
We don't need to provide this constructor and I don't think we should. It enables the undesirable behavior of omitting an error message.